database/schema.sql

```diff
@@ -4,3 +4,4 @@ CREATE TABLE IF NOT EXISTS entries (
   image_url text not null,
   created_at timestamp with time zone default now()
 );
+GRANT ALL PRIVILEGES ON TABLE entries TO postgres;
```

Comment:
Hm, not sure I'm happy about this, in prod the role name isn't `postgres`, let me separate this into another file